reana_commons/config.py

```diff
@@ -43,10 +43,18 @@
 """RabbitMQ connection string."""
 
 STATUS_QUEUE = 'jobs-status'
-"""Name of the queue where to publish/consume from."""
 
 EXCHANGE = ''
-"""RabbitMQ exchange."""
 
 ROUTING_KEY = 'jobs-status'
+
+MQ_DEFAULT_SERIALIZER = 'json'
```

Comment:
Maybe the name for this could be MQ_SERIALIZATION_FORMAT as it refers to the format